engine/src/main/java/org/camunda/bpm/engine/impl/test/TestHelper.java

```diff
@@ -206,17 +205,6 @@ private static String createResourceName(Class< ? > type, String name, String su
     return r.append("." + suffix).toString();
   }
 
-  public static boolean annotationRequiredHistoryLevelCheck(ProcessEngine processEngine, Description description) {
```

Comment:
Do we need to remove this method? The `TestHelper` class is in runtime code, so even though it's in the `impl` package, users might be using this method for their tests.

Reply:
We need to remove this method since the JUnit 4 class `Description` is not on the Quarkus classpath. I also think it is not a good idea to expose this test dependency class at all.